ios/browser/api/bookmarks/bookmark_model_listener_ios.h

```diff
@@ -38,7 +38,8 @@ class BookmarkModelListener : public bookmarks::BookmarkModelObserver {
                          size_t new_index) override;
   void BookmarkNodeAdded(bookmarks::BookmarkModel* model,
                          const bookmarks::BookmarkNode* parent,
-                         size_t index) override;
+                         size_t index,
+                         bool added_by_user) override;
```

Comment:
cc @soner-yuksel in case there's something we need to do here

Reply:
Yeah we do not need / use if bookmark is added by user or not( in this case **not** part is extension or any other source). So I believe we have no need to update anything for  `bookmarkNodeChildrenChanged:`